### PR TITLE
Release 2023-10-17

### PR DESCRIPTION
**Pull request by pavithraes**
Merge `develop` into `main`

### CODE REVIEW

**Comment by pavithraes** [comment]
Everything looks good to me:
- New labs blogs are rendered:
	- [Doctesting for PyData libraries](https://labs-git-release-20231017-quansight.vercel.app/blog/doctesting-for-pydata-libraries)
	- [Hypothesis testing in SymPy](https://labs-git-release-20231017-quansight.vercel.app/blog/integrating-hypothesis-into-sympy) 
- Labs blog updates:
	-  [SciPy array API](https://labs-git-release-20231017-quansight.vercel.app/blog/scipy-array-api) -- the SVG hero image is rendered correctly
- Optimized images across the site look fine (did not verify each one, just a couple)
- [Labs team page](https://labs-git-release-20231017-quansight.vercel.app/team) updates are reflected (Mars, Thomas F, Pamphile removed; and Mateusz, Isuru added)

File: apps/labs/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+October 17, 2023
+
+Via Storyblok
+
+- Remove Thomas Fan, Mars Lee, and Pamphile Roy from the team page (switch Role from `team` to `author`)
+- Added Mateusz Sokół (switch Role from `author` to `team`) and Isuru Fernandoto (newly added) to the team page
```
Comment (bskinn) [inline apps/labs/CHANGELOG.md:6]: Was this part of the typo that needed fixing, @pavithraes?

```suggestion
- Added Mateusz Sokół (switch Role from `author` to `team`) and Isuru Fernando (newly added) to the team page
```

File: apps/labs/posts/integrating-hypothesis-into-sympy.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+- Hypothesis is **_not_** just a bug finder, it helps protect against _future_ bugs. Hypothesis disallows the existence of latent bugs which increases trust in the current implementation of whatever function is being tested. Hypothesis may also reveal weird design patterns.
+
+# Integrating Hypothesis into SymPy
+
+SymPy is an ideal library for property based testing so integration was painless.
+
+### What Has Changed in SymPy?
+
+Hypothesis is now a required _testing_ dependency of SymPy. Property based tests can be created in the `test_hypothesis.py` file in respective tests directories more details in the [new contributor documentation](https://github.com/sympy/sympy/blob/master/doc/src/contributing/new-contributors-guide/writing-tests.md#hypothesis-testing)). An example testing file using hypothesis can be found in [ntheory/tests](https://github.com/sympy/sympy/blob/master/sympy/ntheory/tests/test_hypothesis.py).
```
Comment (bskinn) [inline apps/labs/posts/integrating-hypothesis-into-sympy.md:92]:
```suggestion
Hypothesis is now a required _testing_ dependency of SymPy. Property based tests can be created in a `test_hypothesis.py` file in the appropriate test directories (more details in the [new contributor documentation](https://github.com/sympy/sympy/blob/master/doc/src/contributing/new-contributors-guide/writing-tests.md#hypothesis-testing)). An example testing file using Hypothesis can be found in [ntheory/tests](https://github.com/sympy/sympy/blob/master/sympy/ntheory/tests/test_hypothesis.py).
```

File: apps/labs/posts/integrating-hypothesis-into-sympy.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+### Utilizing Hypothesis in SymPy
+
+Hypothesis was able to find various bugs and code design flaws. Below I will highlight two:
+
+- The [`resultant` function returning incorrect answers](https://github.com/sympy/sympy/issues/25406). While this bug ended up not needing to be resolved, it did reveal the utility in having Hypothesis check [consistency between implementations](https://github.com/sympy/sympy/issues/25406#issuecomment-1652243538) of the same function.
+- There were various issues with the lowest common multiple (LCM) function (notes in [issue#25624](https://github.com/sympy/sympy/issues/25624), [PR#25636](https://github.com/sympy/sympy/pull/25636), and [PR#25517](https://github.com/sympy/sympy/pull/25517#issuecomment-1714474991)), the biggest being when the LCM should make use of the integer implementation vs polynomial implementation (when the defined polynomial is essentially an integer).
```
Comment (bskinn) [inline apps/labs/posts/integrating-hypothesis-into-sympy.md:99]: I'd normally expect to see spaces here

```suggestion
- There were various issues with the lowest common multiple (LCM) function (notes in [issue #25624](https://github.com/sympy/sympy/issues/25624), [PR #25636](https://github.com/sympy/sympy/pull/25636), and [PR #25517](https://github.com/sympy/sympy/pull/25517#issuecomment-1714474991)), the biggest being when the LCM should make use of the integer implementation vs polynomial implementation (when the defined polynomial is essentially an integer).
```

**Review by bskinn — CHANGES_REQUESTED** [review]
General proofing pass on Diane's post.

There's enough here that I think we should hotfix in the release branch, @pavithraes, and then back-merge to `develop` after the merge to `main`.

**Comment by bskinn** [comment]
Friendly reminder, @pavithraes, this release is waiting on review of this proofing pass.

**Comment by bskinn** [comment]
Or, @melissawm, if you have more bandwidth for reviewing these few fixes/suggestions.

**Review by melissawm — APPROVED** [review]
Couldn't find anything beyond your comments, @bskinn - this looks good to me with your changes. Thank you!

**Review by bskinn — APPROVED** [review]
Edits are approved, I believe we are ready to merge to live.

File: apps/labs/posts/doctesting-for-pydata-libraries.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
+---
+title: 'Doctesting for PyData Libraries'
+authors: [sheila-kahwai]
+published: September 30, 2023
```
Comment (bskinn) [inline apps/labs/posts/doctesting-for-pydata-libraries.md:4]:
```suggestion
published: October 30, 2023
```

File: apps/labs/posts/integrating-hypothesis-into-sympy.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
+---
+title: 'Integrating Hypothesis into SymPy'
+authors: [diane-tc]
+published: September 20, 2023
```
Comment (bskinn) [inline apps/labs/posts/integrating-hypothesis-into-sympy.md:4]:
```suggestion
published: October 30, 2023
```

**Review by bskinn — COMMENTED** [review]
Update pub dates

**Pull request merged**